### DISCUSSION TITLE
Missing voltage from WSDCGQ11LM

### DISCUSSION THREAD

**Discussion by sineverba** [Koenkk/zigbee2mqtt#13735] [Devices]
Hi there,
from about HA version 2022.01 (If I remember correctly), I'm missing the VOLTAGE of [WSDCGQ11LM](https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html#xiaomi-wsdcgq11lm)

In Z2M control panel I have

![a](https://user-images.githubusercontent.com/2634842/187079620-6709a7ca-bce7-4b5a-a220-48407f2a1458.png)

and in Home Assistant

![b](https://user-images.githubusercontent.com/2634842/187079628-8015e123-cd4b-4653-ab9d-73db87197a4c.png)

Version of Home Assistant is 2022.8.8 and Z2M is 1.26.0, both as Docker containers.

Before of an update I had it, 'cause till today I have in my lovelace...

![c](https://user-images.githubusercontent.com/2634842/187079700-c286d2ab-b28f-4ea9-bc28-9d12a2766764.png)

No configurations done in HomeAssistant, no missing entities unused.

Thank you!

**Comment by jerrm** [comment]
It's probably there but disable. Look at the HA device page and expand the "not shown" entities.

**Reply by sineverba** [reply]
Yes, thank you!
